lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,111 @@
+require 'rake'
+
+ORGANIZATION_ID = 13856
+
+# Removes group members from Teem that are no longer in Roles Management.
+#
+# +token+ is a string containing the access token used to authorize against Teem API
+# +teem_users_to_remove+ is an array of strings containing users in Teem that need to be removed
+# +all_user_ids+ is an array of strings containing a mapping from user email to id in Teem
+# +group_id+ is an integer containing the group id
+def remove_teem_group_members(token, teem_users_to_remove, all_user_ids, group_id)
+  teem_users_to_remove.each do |email|
+    user_id = all_user_ids[email]
+
+    user = Teem.teem_get_user(token, user_id)
+    user_groups = user["group_ids"]
+    user_groups.delete(group_id)
+    
+    Teem.teem_update_user_groups(token, user_groups, user_id)
+  end
+end
+
+# Adds group members to Teem that are in Roles Management.
+#
+# +token+ is a string containing the access token used to authorize against Teem API
+# +teem_users_to_add+ is an array containing users in Teem that need to be added
+# +all_user_ids+ is an array of strings containing a mapping from user email to id in Teem
+# +group_id+ is an integer containing the group id
+# +role+ is a json object containing a roles information
+def add_teem_group_members(token, teem_users_to_add, all_user_ids, group_id, role)
+  teem_users_to_add.each do |email|
+    user_id = all_user_ids[email]
+
+    if user_id
+      Teem.teem_update_user_groups(token, group_id, user_id)
+    else
+      role.members.each do |member|
+        if member["email"].downcase == email.downcase
+          first_name = member["first"]
+          last_name = member["last"]
+
+          Teem.teem_create_user(token, ORGANIZATION_ID, email, first_name, last_name)
+        end
+      end
+      Teem.teem_update_user_groups(token, group_id, user_id)
+    end
+  end
+end
+
+namespace :teem do
+  desc 'Sync RM users to Teem'
+  task sync_users: :environment do
+    require 'teem.rb'
+
+    token = Teem.teem_authorize()
```

Comment:
It's convention in Ruby to omit parentheses whenever possible, e.g. `token = Teem.teem_authorize`. It'll still be a function call. It takes some getting used to.